introspection.go

```diff
@@ -266,6 +266,14 @@ mutation operations.`,
 					return nil, nil
 				},
 			},
+			"subscriptionType": &Field{
+				Description: `If this server support subscription, the type that ' +
+                   'subscription operations will be rooted at.`,
+				Type: __Type,
+				Resolve: func(p ResolveParams) (interface{}, error) {
+					return nil, nil
+				},
+			},
```

Comment:
I think we need to implement the subscription feature but before that we need a working graphql server. Without this the introspection query, graphiql and relay fails and we couldn't use our frontend without this. Maybe we should let the user's know that the subscription feature is not implemented but type exists in our system for compatibility's sake.